app/assets/stylesheets/pages/_home.scss

```diff
@@ -13,16 +13,20 @@
 .mainh6 {
   font-family: 'Averia Gruesa Libre';
   font-size: 32px;
-  margin-top: 10px;
+  color: #FFFFFF;
 }
 
 .mainh1 {
   font-size: 120px;
   font-weight: bold;
   font-family: 'Lato';
-  margin-top: 80px;
+  margin-top: 30px;
+  color: #FFFFFF;
 }
 
+body {
```

Comment:
@Kameronjan you can remove this from your scss. I put background color and background image styling in it's own scss file.